src/main/java/com/exasol/adapter/installer/JdbcDriverLocalFileProvider.java

```diff
@@ -0,0 +1,23 @@
+package com.exasol.adapter.installer;
+
+/**
+ * This class provide a locally stored JDBC driver JAR.
+ */
+public class JdbcDriverLocalFileProvider implements JdbcDriverJarProvider {
+    private final Jar jar;
+
+    /**
+     * Instantiates a new {@link JdbcDriverLocalFileProvider}
```

Comment:
I'm all for keeping the Java standard about javadocs. I would suggest if we want to do it this way, then change the other javadocs as well, if not keep it to the non-standard way.

```suggestion
     * Instantiate a new {@link JdbcDriverLocalFileProvider}
```